vars/util.groovy

```diff
@@ -211,22 +211,36 @@ GIT INFORMATION REPORT
 
 def getNextVersion(String version, String type, String suffix = 'SNAPSHOT') {
     assert ['major', 'minor', 'micro'].contains(type)
+    Integer[] versionSplit = parseVersion(version)
+    if (versionSplit != null){
+        int majorVersion = versionSplit[0] + (type == 'major' ? 1 : 0)
+        int minorVersion = versionSplit[1] + (type == 'minor' ? 1 : 0)
+        int microVersion = versionSplit[2] + (type == 'micro' ? 1 : 0)
+        return "${majorVersion}.${minorVersion}.${microVersion}${suffix ? '-' + suffix : ''}"
+    } else {
+        return null
+    }
+}
+
+Integer[] parseVersion(String version){
```

Comment:
can you please add comments about what the method is doing. Thanks.

Reply:
updated. let me know